src/AzSK.ADO/Framework/Listeners/UserReports/WriteSummaryFile.ps1

```diff
@@ -228,14 +228,14 @@ class WriteSummaryFile: FileOutputBase
 					}
 					#if($anyFixableControls)
 					#{
-					if($item.ControlItem.FixControl)
-					{
-						$csvItem.SupportsAutoFix = "Yes";
-					}
-					else
-					{
-						$csvItem.SupportsAutoFix = "No";
-					}
+					# if($item.ControlItem.FixControl)
```

Comment:
You can use multiline comment here as below
<#
Multiple lines of code to comment
#>